src/main/resources/init_db.sql

```diff
@@ -1,13 +1,73 @@
 CREATE TABLE public.items
 (
-    id bigint NOT NULL,
-    name "char"[] NOT NULL,
-    price money[] NOT NULL,
-    PRIMARY KEY (id)
+    id integer NOT NULL DEFAULT nextval('items_id_seq'::regclass),
+    name character varying(255) COLLATE pg_catalog."default" NOT NULL,
+    price double precision NOT NULL,
+    CONSTRAINT items_pkey PRIMARY KEY (id)
 )
-WITH (
-    OIDS = FALSE
-);
+    WITH (
+        OIDS = FALSE
+    )
+    TABLESPACE pg_default;
 
 ALTER TABLE public.items
     OWNER to postgres;
+
+CREATE TABLE public.users
+(
+    id integer NOT NULL DEFAULT nextval('users_id_seq'::regclass),
+    login character varying(255) COLLATE pg_catalog."default" NOT NULL,
+    password character varying(255) COLLATE pg_catalog."default" NOT NULL,
+    name character varying(255) COLLATE pg_catalog."default",
+    surname character varying(255) COLLATE pg_catalog."default",
+    token character varying(255) COLLATE pg_catalog."default",
+    CONSTRAINT users_pkey PRIMARY KEY (id)
+)
+    WITH (
+        OIDS = FALSE
+    )
+    TABLESPACE pg_default;
+
+ALTER TABLE public.users
+    OWNER to postgres;
+
+CREATE TABLE public.orders
+(
+    id integer NOT NULL DEFAULT nextval('orders_id_seq'::regclass),
+    user_id integer NOT NULL,
+    CONSTRAINT orders_pkey PRIMARY KEY (id),
+    CONSTRAINT orders_users_fk FOREIGN KEY (user_id)
+        REFERENCES public.users (id) MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+)
+    WITH (
+        OIDS = FALSE
+    )
+    TABLESPACE pg_default;
+
+ALTER TABLE public.orders
+    OWNER to postgres;
```

Comment:
@SkaterWeather is it required?

Reply:
last time i didn't make any changes to init_db.sql, because i was thinking to create backup file.